### DISCUSSION TITLE
How to switch freely between multiple audio output devices?

### DISCUSSION THREAD

**Discussion by Lyneon** [LizardByte/community#470] [Sunshine] [answered]
I use Sunshine on Windows PC and Moonlight on Android tablet.
There are situations where I want to temporarily change the audio output to my PC's Speaker. But when I try to change the audio output from Streaming Speaker to PC's Speaker, Sunshine appears to reset the audio device back to the Steaming Speaker :( it gives some logs:
`[2024:06:16:12:41:04]: Info: Resetting sink to [virtual-Stereo{0.0.0.00000000}.{d086e213-4dfd-4503-b00a-1cf5bf300fcc}] after default changed`
`[2024:06:16:12:41:04]: Info: Reinitializing audio capture`
Although I can specify the virtual audio output device in the Audio/Video in the WebUI config page to use the PC speakers, then I can't change it back to Straming Speaker anymore.

So I would like to ask, is there a way to switch freely between multiple audio output devices without having to change the  Audio config  in Sunshine every time? Thanks a lot.

**Comment by Lyneon** [comment, marked as answer]
It seems that turning on the option to play sound on the computer at the same time in Moonlight will do the trick. Sorry to interrupt

**Reply by onlyesterday** [reply]
Where is the option to play sound in moonlight and sunshine? Thx.

**Reply by Lyneon** [reply]
In Moonlight, Host Settings -> play audio on PC
![Screenshot_20240701-223239_Moonlight](https://github.com/LizardByte/Sunshine/assets/69835744/afcb4849-dff2-4e60-b3e2-0b764ab67d48)

**Reply by Lyneon** [reply]
In this way, you can play sound on Moonlight and Sunshine the same time. Then you can mute mobile so that it only play sound on PC, or mute your PC so that it only play sound on mobile.

**Reply by onlyesterday** [reply]
Thank you. Finally found that option in Moonlight. 😂

**Reply by Lyneon** [reply]
Congratulations
